src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,53 @@
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+  currentTodo: Todo | null;
+  setCurrentTodo: (todo: Todo) => void;
+}
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  currentTodo,
+  setCurrentTodo,
+}) => {
+  const handleClick = () => {
+    setCurrentTodo(todo);
+  };
+
+  const { title, id, completed } = todo;
+
+  return (
+    <tr data-cy="todo" className="">
+      <td className="is-vcentered">{id}</td>
+      {todo.completed ? (
+        <td className="is-vcentered">
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        </td>
+      ) : (
+        <td className="is-vcentered" />
+      )}
+      <td className="is-vcentered is-expanded">
+        <p className={completed ? 'has-text-success' : 'has-text-danger'}>
```

Comment:
Use the classnames library rules for add classes with condition, fix it everywhere